packages/odyssey-react-labs/src/datePickerTheme.tsx

```diff
@@ -76,20 +77,60 @@ const todayDotStyles: StateStyles = {
   }),
 };
 
+const yearStyles: StateStyles = {
+  default: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.primary,
+  }),
+  disabled: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.disabled,
+  }),
+  hover: ({ theme }) => ({
+    backgroundColor: theme.palette.grey[100],
+  }),
+  hoverSelected: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.lighter,
+    color: theme.palette.primary.main,
+  }),
+  outsideOfMonth: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.secondary,
+  }),
+  selected: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.primary.main,
+    fontWeight: theme.typography.fontWeightBold,
+  }),
+};
+
+const yearCheckStyles: StateStyles = {
+  default: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.main,
+    content: '""',
+    height: `${(16 / 16) * (16 / 14)}rem`,
```

Comment:
Similar comment as above here. Could this be `1em` or `theme.typography.body.fontSize`?

Reply:
I made it relative to the `fontSize`.

```suggestion
    height: theme.typography.body1.fontSize,
```

Reply:
Actually, I changed this to `h6` because `1rem` is actually `14px` rather than `16px` to match the text. `FontScale2`.

Reply:
Totally cool to use `fontSize` variables here. Another option I use sometimes is `theme.typography.ui.lineHeight`, which is unitless. You can tag an `em` on there to make it scale with the local font-size (which gives you 16px with 14px type).